### PR TITLE
fix(website): rework footer links formatting

### PR DESCRIPTION
**Pull request by wolfderechter**
Rework footer links formatting regarding issue #349

### CODE REVIEW

**Review by dantaik — DISMISSED** [review]

**Comment by d1onys1us** [comment]
image
image

@wolfderechter i noticed that this caused the dots to be un-centered now in the normal landscape view

File: packages/website/src/css/custom.css [hunk 1]
```diff
@@ -117,3 +117,14 @@
...
+
+.footer__links {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 100%;
+}
+
+.footer__link-item {
+  margin-right: 16px;
```
Comment (d1onys1us) [inline packages/website/src/css/custom.css:129]: think it might be due to this line

**Comment by wolfderechter** [comment]
@d1onys1us you are correct, I have missed that initially. Fixed it now

**Comment by d1onys1us** [comment]
> @d1onys1us you are correct, I have missed that initially. Fixed it now

thank you. i also notice now that the dots are shifted up from center as well.

prev:
image
after:
image

**Comment by wolfderechter** [comment]
Ahh the devil is in the detail! Okay should be all good now

**Review by dionysuzx — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Comment by d1onys1us** [comment]
@all-contributors please add @wolfderechter code,doc

**Pull request merged**